model/rulefmt/testdata/test_aliases.yaml

```diff
@@ -0,0 +1,50 @@
+groups:
+  - name: my-group-name
+    interval: 30s # defaults to global interval
+    rules:
+      - alert: &alertname HighAlert
```

Comment:
I was curious to see whether this worked with a YAML merge, as in a map key of `<<:`, it turns out it does before and after this change. But I'll push a test to make sure it carries on working too as I don't see us test YAML merges anywhere. (I know there are some other places in unit tests where they may not work currently too.)

Reply:
I didn't think about YAML merge here indeed, thanks for the tests!